README.md

```diff
@@ -129,6 +136,8 @@ See [C++ Programmer's Toolbox](#c-programmers-toolbox) for details.
 1.  **Copy Constructor and Move Constructor**
     *   If you provide copy constructor and/or move constructor,
         provide the corresponding `operator=` overload as well.
+    *   Class should be disable copy/move by default.
```

Comment:
I think this statement is too strong. I think for a style guide/primer it is much more valuable if we could explain trade-offs  in detail, like what you did for unnamed namespace :)

Copy and move constructors can be very useful, and the programmer should use their best judgement when implementing/disabling them.